File: launchdarkly/python/src/PostApprovalRequestReviewExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    post_approval_request_review_request = models.PostApprovalRequestReviewRequest(
        kind="approve",
        comment="Looks good, thanks for updating",
    )

    try:
        response = api.ApprovalsApi(api_client).post_approval_request_review(
            id=None,
            post_approval_request_review_request=post_approval_request_review_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Approvals#post_approval_request_review: %s\n" % e)

File: launchdarkly/ruby/src/PostApprovalRequestReviewExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

post_approval_request_review_request = LaunchDarklyClient::PostApprovalRequestReviewRequest.new
post_approval_request_review_request.kind = "approve"
post_approval_request_review_request.comment = "Looks good, thanks for updating"

begin
    response = LaunchDarklyClient::ApprovalsApi.new.post_approval_request_review(
        nil, # id
        post_approval_request_review_request,
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling ApprovalsApi#post_approval_request_review: #{e}"
end
